tools/pd-ctl/main.go

```diff
@@ -59,10 +59,6 @@ func main() {
 	flag.CommandLine.ParseErrorsWhitelist.UnknownFlags = true
 	flag.Parse()
 
-	if help {
```

Comment:
Is it appropriate to output subcommands option in help ?

Reply:
It should be. The original implementation I think is not so correct. We use `pflag`  and `cobra` together to parse args, but `cobra` self is enough in fact. The `--help` message of pflag we set is too simple and override the help message from cobra which is more friendly.

```
> pd-ctl --help // print help message for pd-ctl
> pd-ctl store --help // print help message for store
```

Reply:
```
./pd-ctl store --help                                                                       

manipulate or query stores

Usage:
  pd-ctl store [command] [flags]
  pd-ctl store [command]

Available Commands:
  delete           delete the store
  label            set a store's label value
  limit            set a store's rate limit
  remove-tombstone remove tombstone record if only safe
  weight           set a store's leader and region balance weight

Flags:
      --jq string   jq query

Global Flags:
      --cacert string   The path of file that contains list of trusted SSL CAs.
      --cert string     The path of file that contains X509 certificate in PEM format.
  -d, --detach          Run pdctl without readline. (default true)
  -h, --help            Help message.
  -i, --interact        Run pdctl with readline.
      --key string      The path of file that contains X509 key in PEM format.
  -u, --pd string       pd address (default "http://127.0.0.1:2379")
  -V, --version         Print version information and exit.

Use "pd-ctl store [command] --help" for more information about a command.
```

Reply:
It seem output global flags too.

Reply:
And it echo other redundant flags `--jq string   jq query`.

Reply:
> It seem output global flags too.

Yes, it should print the local and global flags.

> And it echo other redundant flags `--jq string jq query`.

There is only one `--jq` here, why do you think it is redundant?